File: app/src/main/java/com/karim/booksapp/data/models/pojos/PanelizationSummary.java

package com.karim.booksapp.data.models.pojos;

import com.squareup.moshi.Json;

public class PanelizationSummary {

    @Json(name = "containsEpubBubbles")
    public Boolean containsEpubBubbles;
    @Json(name = "containsImageBubbles")
    public Boolean containsImageBubbles;

}

File: app/src/main/java/com/karim/booksapp/data/models/pojos/PanelizationSummary.kt
package com.karim.booksapp.data.models.pojos

import com.squareup.moshi.Json

class PanelizationSummary {

    @Json(name = "containsEpubBubbles")
    var containsEpubBubbles: Boolean? = null
    @Json(name = "containsImageBubbles")
    var containsImageBubbles: Boolean? = null

}
